contributing/code/standards.rst

```diff
@@ -172,6 +172,10 @@ Documentation
 
 * Add PHPDoc blocks for all classes, methods, and functions;
 
+* Group annotations together so that annotations of the same type immediately
```

Comment:
this rule is in the "Documentation" section. This means it's focused on the PHPdoc annotations